py/server/deephaven/table.py

```diff
@@ -2087,9 +2087,10 @@ def slice(self, start: int, stop: int) -> Table:
     def slice_pct(self, start_pct: float, end_pct: float) -> Table:
         """Extracts a subset of a table by row percentages.
 
-        Returns a subset of table in the range [startPercentInclusive * sizeOfTable , endPercentExclusive * sizeOfTable).
+        Returns a subset of table in the range [start_pct * size_of_table , end_pct * size_of_table).
         For example, for a table of size 10, slicePct(0.1, 0.7) will return a subset from the second row to the seventh
-        row. The percentage arguments must be in [0,1], otherwise the function returns an error.
+        row. Similarly, slicePct(0, 1) would return the entire table (because row positions run from 0 to size-1).
+        The percentage arguments must be in range [0,1], otherwise the function returns an error.
```

Comment:
The description is still using the java function name.  Check carefully for other places where java is used.